pkg/config/server_config.go

```diff
@@ -1,12 +1,13 @@
 package config
 
 type HttpServerConfig struct {
-	Hostname   string `mapstructure:"hostname"`
-	BaseUrl    string `mapstructure:"baseUrl"`
-	TlsEnabled bool   `mapstructure:"tlsEnabled"`
-	CertFile   string `mapstructure:"certFile"`
-	KeyFile    string `mapstructure:"keyFile"`
-	Port       int    `mapstructure:"port"`
+	Hostname      string `mapstructure:"hostname"`
+	BaseUrl       string `mapstructure:"baseUrl"`
+	TlsEnabled    bool   `mapstructure:"tlsEnabled"`
+	CertFile      string `mapstructure:"certFile"`
+	KeyFile       string `mapstructure:"keyFile"`
+	Port          int    `mapstructure:"port"`
+	SessionSecret string `mapstructure:"sessionSecret"`
```

Comment:
@jules evaluate how secure this method is for a production environment. Is there a way to implement key rotation properly?